### PR TITLE
configure.ac: removed GObject Introspection dependency from owr-java

### PR DESCRIPTION
**Pull request by Rugvip**
We don't really need introspection for owr-java, just the Owr-1.0.gir file

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -144,7 +144,7 @@ AC_HELP_STRING(
 esac],[enable_owr_java=no])
 AC_MSG_RESULT([$enable_owr_java])
 if test "x$enable_owr_java" = xyes; then
-  if test "x$found_introspection" != xyes; then
+  if test ! find ${prefix}/.. -name Owr-0.1.gir; then
```
Comment (sdroege) [inline configure.ac]: Is this going to result in a syntax error if a patch with a space is found, or multiple paths?

**Comment by Rugvip** [comment]
I couldn't figure out a good check for the .gir file, so let's just remove the gobject introspection dependency for now

**Comment by sdroege** [comment]
I meant just adding some " around there btw :)

**Comment by Rugvip** [comment]
I didn't see your comment before I removed it, the problem was that the change was making assumptions about the folder structure of the different build targets, which I guess would be pretty broken.

**Comment by sdroege** [comment]
True

**Comment by nirbheek** [comment]
:+1:

**Comment by superdump** [comment]
Merged.

**Pull request closed without merging**